faker/providers/user_agent/__init__.py

```diff
@@ -13,20 +13,39 @@
 class Provider(BaseProvider):
     user_agents = (
         'chrome', 'firefox', 'internet_explorer', 'opera', 'safari',
+        'microsoft_edge',
     )
 
     windows_platform_tokens = (
         'Windows 95', 'Windows 98', 'Windows 98; Win 9x 4.90', 'Windows CE',
         'Windows NT 4.0', 'Windows NT 5.0', 'Windows NT 5.01',
         'Windows NT 5.1', 'Windows NT 5.2', 'Windows NT 6.0', 'Windows NT 6.1',
-        'Windows NT 6.2',
+        'Windows NT 6.2', 'Windows NT 6.3', 'Windows NT 10.0',
     )
 
     linux_processors = ('i686', 'x86_64',)
 
     mac_processors = ('Intel', 'PPC', 'U; Intel', 'U; PPC',)
 
-    langs = ('en-US', 'sl-SI', 'it-IT',)
+    # Language codes from ISO 639-1
+    langs = (
```

Comment:
from what I've seen, user agent strings usually use a locale (eg: `en-us`) rather than languages.

Maybe this can be refactored to make use of https://github.com/joke2k/faker/blob/master/faker/providers/misc/__init__.py#L71, and you can exapnd the list there?